enterprise/api/v1/serializers.py

```diff
@@ -779,6 +780,7 @@ class Meta:
         default=[],
         child=serializers.EmailField()
     )
+    enable_compression = serializers.BooleanField()
```

Comment:
Why do we need to add `enable_compression` here? wouldn't this be added automatically by DRF?

Reply:
yes, you are right. I will update this.